[BUG] Installing 2.5.0 on Vercel fails

**Describe the bug**
Just started out using inngest so i'm not that familar with it yet. So could be i'm missing something here... But my first deploys to Vercel are failing and I don't really know why that is. 

When I switch to using 2.4.1 it works fine.

When I install and build using 2.5.0 locally, it all works fine.

See Vercel logs below.

**To Reproduce**
Steps to reproduce the behavior:

1. Use 2.5.0
2. Deploy to Vercel, error
3. Use 2.4.1
4. Deploy to Vercel, no error

**Expected behavior**
No errors during build on Vercel on 2.5.0

**Code snippets / Logs / Screenshots**
Happens during `npm install`:

```
01:18:29.223 | npm ERR! code 254
01:18:29.223 | npm ERR! path /vercel/path0/node_modules/inngest
01:18:29.224 | npm ERR! command failed
01:18:29.224 | npm ERR! command sh -c npx only-allow pnpm
01:18:29.224 | npm ERR! npm ERR! code ENOENT
01:18:29.224 | npm ERR! npm ERR! syscall lstat
01:18:29.224 | npm ERR! npm ERR! path /vercel/path0/lib
01:18:29.224 | npm ERR! npm ERR! errno -2
01:18:29.224 | npm ERR! npm ERR! enoent ENOENT: no such file or directory, lstat '/vercel/path0/lib'
01:18:29.224 | npm ERR! npm ERR! enoent This is related to npm not being able to find a file.
01:18:29.224 | npm ERR! npm ERR! enoent
01:18:29.225 | npm ERR!
01:18:29.225 | npm ERR! npm ERR! A complete log of this run can be found in:
01:18:29.225 | npm ERR! npm ERR!     /vercel/.npm/_logs/2023-08-19T23_18_27_175Z-debug-0.log
01:18:29.225 |  
01:18:29.225 | npm ERR! A complete log of this run can be found in:
01:18:29.225 | npm ERR!     /vercel/.npm/_logs/2023-08-19T23_17_59_611Z-debug-0.log
01:18:29.266 | Error: Command "npm install --prefix=../.." exited with 254
```

**System info (please complete the following information):**
- OS: Mac
- npm package Version: 2.5.0
- Framework: Next.JS
- Platform: Vercel

**Additional context**
-

--- a/.changeset/beige-hotels-deliver.md
+++ b/.changeset/beige-hotels-deliver.md
@@ -1,0 +1,5 @@
+---
+"inngest": patch
+---
+
+Remove `preinstall` script from `inngest` package causing errors when consuming the package

--- a/packages/inngest/package.json
+++ b/packages/inngest/package.json
@@ -8,7 +8,6 @@
     "registry": "https://registry.npmjs.org"
   },
   "scripts": {
-    "preinstall": "npx only-allow pnpm",
     "prebuild": "pnpm run pb:version",
     "pb:version": "genversion --semi --double --es6 ./src/version.ts",
     "build": "pnpm run clean && tsc --project tsconfig.build.json",

